src/enforcer.js

```diff
@@ -463,19 +548,25 @@ export class TrustedTypesEnforcer {
    * @return {*}
    */
   enforceTypeInScriptNodes_(context, checkParent, originalFn, ...args) {
+    if (!this.windowObject.Node || !this.windowObject.document) return;
```

Comment:
You already check for Node in one place, when installing this function. In the other you check for Element only. Add a check for Node there, and remove from here. When a hooking function is already installed, I would prefer if it threw when any strange environment issue is detected.

Reply:
I don't get this part.

>  In the other you check for Element only. Add a check for Node there, and remove from here.

Elaborate pls.

Reply:
Done.